src/commands/set-policy.yml

```diff
@@ -0,0 +1,68 @@
+description: "Set policy to pull a repository with secondary ids"
+parameters:
+
+  secondary-account-ids:
+    description: A comma-separated string containing secondary-account-ids
```

Comment:
The "env_var_name" type should not allow commas. This may have to be left as a string if comma-separated is required.

Reply:
Ahh and I just realized this means the env var value is comma-separated. My mistake.

Reply:
Yes, it is an env_var with value comma-separated.
I didn't know if I use an env_var or a string.